src/formats/json-schema.ts

```diff
@@ -0,0 +1,41 @@
+import { JSONSchema4, JSONSchema6, JSONSchema7 } from 'json-schema';
+import { isObject } from '../utils/isObject';
+
+type JSONSchema = JSONSchema4 | JSONSchema6 | JSONSchema7;
+
```

Comment:
Is any in the spec, or some shortcut/trick we use, or a mistake?

Reply:
@philsturgeon 
Yeah, it's specified in the following doc https://tools.ietf.org/html/draft-zyp-json-schema-03#section-5.1
The newer doc https://tools.ietf.org/html/draft-zyp-json-schema-04#section-3.5 does not have it specified though.

Reply:
@P0lip i think you can safely ignore anything in draft 3, that's 9 years old and draft 4 is the oldest thing we want to pay attention to as its the basis of OpenAPI 2, or draft 5 because OpenAPI 3. Folks are either using OpenAPI, or they're using modern JSON Schema (6 or 7). If they are on 3 then... erf? 

It doesn't hurt to have this in here, but lets not consider draft 3 for any other code we work on.